### PR TITLE
Manage VM: #start and #stop

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: app/models/foreman_google/google_compute.rb [hunk 1]
```diff
@@ -30,9 +30,11 @@ def status
     end
 
     def start
+      @client.start(@zone, @identity)
```
Comment (ezr-ondrej) [inline app/models/foreman_google/google_compute.rb:33]: should we add guard for `if persisted?` :)
Not strictly necessary, but would be good manners IMHO

Comment (stejskalleos) [reply app/models/foreman_google/google_compute.rb:33]: Yeah probably, should we do it also for the `stop` method?

**Review by ezr-ondrej — APPROVED** [review]
LGTM, adding guards would be good to have :+1:

**Comment by ezr-ondrej** [comment]
eh :shrug: let me do so in a follow-up :)

**Pull request merged**